roles/ora-host/defaults/main.yml

```diff
@@ -72,6 +72,62 @@ oracle_required_rpms:
   - xorg-x11-utils
   - xorg-x11-xauth
 
+oracle_required_rpms_el8:
```

Comment:
Apologies if I missed a note earlier, but the documentation (https://google.github.io/bms-toolkit/user-guide.html#target-server-requirements) still mentions only OL7 being supported;  should that be updated here, or are there other changes required to support this combination?  Thanks!

Reply:
Thanks for the suggestion, yes, I think this PR is a good place to include that update.
Will update `bms-toolkit/docs/user-guide.md` and refresh the branch.